api/info/info.go

```diff
@@ -50,8 +51,22 @@ func (h *Handler) Handle(w http.ResponseWriter, r *http.Request) error {
 	if err != nil {
 		return err
 	}
+
+	var profile *config.SigningProfile
+
+	policy := h.sign.Policy()
+	if policy != nil && policy.Profiles != nil && req.Profile != "" {
+		profile = policy.Profiles[req.Profile]
+	}
+
+	if profile == nil && policy != nil {
+		profile = policy.Default
+	}
+
```

Comment:
Should verify that profile is not nil before using its fields below. It shouldn't be nil in normal operation, so you can fail with an error if it is.